### DISCUSSION TITLE
买了一年的蓝灯，还是免费版

### DISCUSSION THREAD

**Discussion by AIR-SORA** [getlantern/lantern-forum-cn#678] [Q&A]
357369051@qq.com 这个账号买的，一直是免费版，用授权码，邮箱也收不到。

**Comment by AIR-SORA** [comment]
@Mike777Khoo  能解决一下吗？

**Reply by MaxMKMax** [reply]
@AIR-SORA 您好. 我们会帮您解决问题. 请您放心. 为了保护您的隐私, 请您直接在这个链接上提交您的资料就好了. 支付单据和邮箱. 
后续您可以在GitHub这个帖子和我跟进如果您的问题还没解决. 谢谢您!
https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ

*备注: 如果各位看到这个链接, 请您依然开个帖子说明您的问题. 这样我们才能发现到您的问题帮您跟进. 不要只点连接提交问题而不开帖子. 帖子上不需要个人资料了. 邮箱和支付单据放在链接里就好了. 谢谢!

**Reply by AIR-SORA** [reply]
已经提交了，请尽快解决，谢谢。
